addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -467,7 +467,7 @@
                 <field name="name" widget="text"/>
             </field>
             <xpath expr="//div[hasclass('col_left')]" position="after">
-                <div class="o_hr_leave_column col_right col-md-6 col-12">
+                <div class="o_hr_leave_column col_right col-md-6 col-12 py-3 bg-200">
```

Comment:
Same comment about padding, relying on the form_sheet padding could be sufficient

Reply:
I need to apply this padding since we use a negative margin to place the gray background on the right element